core/src/main/resources/hudson/slaves/CommandLauncher/help-command_ja.html

```diff
@@ -4,29 +4,29 @@
 
   <p>
   この項目にコマンドが指定されたとき、マスターでこのコマンドが実行されます。そして、
-  Jenkinsは、その実行コマンドが正しいスレーブマシーン上で<tt>slave.jar</tt>を起動することを想定しています。
+  Jenkinsは、その実行コマンドが正しいスレーブマシーン上で<tt>agent.jar</tt>を起動することを想定しています。
 
   <p>
-  <tt>slave.jar</tt>のコピーが、<tt>jenkins.war</tt>の中の<a href="${rootURL}/jnlpJars/slave.jar"><tt>WEB-INF/slave.jar</tt></a>にあります。
+  <tt>agent.jar</tt>のコピーが、<tt>jenkins.war</tt>の中の<a href="${rootURL}/jnlpJars/agent.jar"><tt>WEB-INF/agent.jar</tt></a>にあります。
```

Comment:
There is no `WEB-INF/agent.jar` or slave.jar anymore. A while ago @jglick has removed these duplications, so there is only WEB-INF/lib/remoting-*.jar

Ideally we need to remove the reference from all translations

Reply:
indeed. I'll need assistance to fix core/src/main/resources/hudson/slaves/CommandLauncher/help-command_ru.html and core/src/main/resources/hudson/slaves/CommandLauncher/help-command_ja.html

Reply:
suggested Russian translation.
We need @kohsuke or @ikedam for the Japanese one

Reply:
Created jenkinsci/command-launcher-plugin#2